notebooks/official/model_monitoring/model_monitoring.ipynb

```diff
@@ -336,13 +336,14 @@
       "source": [
```

Comment:
<p>Remove HW version and replace with sometjhing like:</p><p> </p><p>gs://[cloud-storage-monitoring-bucket]/*/*</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/859/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>I'm trying to show a pre-run version. The data won't be in their bucket for hours so I'm using the hardcoded version to explain the contents and I explain that in the text.</p>